### ISSUE TITLE
UNKNOWN ERROR! TypeError: Matrix.service.firebase.user.watchAppInstall is not a function

### ISSUE THREAD

**Issue by DieterKoblenz** [matrix-io/matrix-os#21]
I am getting the following error on the most recent matrix-os:

`Installed Apps:
Local Apps: [ 'clock.matrix' ]
Local / Installed Δ 1
FIREBASE WARNING: Exception was thrown by user callback. TypeError: Matrix.service.firebase.user.watchAppInstall is not a function
    at setupFirebaseListeners (/home/pi/matrix-os/index.js:257:37)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
    at replenish (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1030:17)
    at iterateeCallback (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1015:17)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:988:16
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3850:13
    at apply (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:41:25)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:76:12
    at syncApps (/home/pi/matrix-os/index.js:232:9)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
Uncaught exception:  [TypeError: Matrix.service.firebase.user.watchAppInstall is not a function] TypeError: Matrix.service.firebase.user.watchAppInstall is not a function
    at setupFirebaseListeners (/home/pi/matrix-os/index.js:257:37)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
    at replenish (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1030:17)
    at iterateeCallback (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1015:17)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:988:16
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3850:13
    at apply (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:41:25)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:76:12
    at syncApps (/home/pi/matrix-os/index.js:232:9)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
UNKNOWN ERROR! TypeError: Matrix.service.firebase.user.watchAppInstall is not a function
    at setupFirebaseListeners (/home/pi/matrix-os/index.js:257:37)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
    at replenish (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1030:17)
    at iterateeCallback (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:1015:17)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:988:16
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3850:13
    at apply (/home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:41:25)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:76:12
    at syncApps (/home/pi/matrix-os/index.js:232:9)
    at /home/pi/matrix-os/node_modules/matrix-firebase/node_modules/async/dist/async.js:3845:9
Unable to clean, exitting...
`

**Comment by brianofrokk3r** [comment]
@DieterKoblenz -- didn't see this until now, looks like all you need to do is... 

This is happening because one of the `node_modules` is not updated. Try this:
```
rm -rf node_modules
npm install
```

**Note:** If you're starting fresh...
```
https://github.com/matrix-io/matrix-os.git
git submodule update --init 
npm install
```

**Comment by brianofrokk3r** [comment]
@DieterKoblenz let us know if you have any issues.

**Comment by brianofrokk3r** [comment]
Resolved in [v0.6.0](https://github.com/matrix-io/matrix-os/releases/tag/v0.6.0)

**Comment by brianofrokk3r** [comment]
@DieterKoblenz please let us know if you have any additional issues. Please pull latest version from master.
